app/backend/wells/views.py

```diff
@@ -151,7 +151,7 @@ class WellListAPIView(ListAPIView):
 
     permission_classes = (DjangoModelPermissionsOrAnonReadOnly,)
     model = Well
-    queryset = Well.objects.all()
+    queryset = Well.objects.exclude(well_publication_status='Unpublished')
```

Comment:
If a logged in user wants to edit a well, to make it published, the user would have to be able to find it. I think we should only apply this filter to users who aren't logged in with well edit rights.